app/bundles/LeadBundle/Helper/CustomFieldHelper.php

```diff
@@ -60,7 +71,11 @@ public static function fieldValueTransfomer(array $field, $value)
                     return null;
                 }
 
-                $dtHelper = new DateTimeHelper($value, null, 'local');
+                if (!($value instanceof \DateTimeInterface) && !is_string($value)) {
+                    throw new \InvalidArgumentException('Wrong type given. String or DateTimeInterface expected.');
+                }
+
+                $dtHelper = self::getDateTimeHelper($value);
```

Comment:
I don't think this will work for the second date value that will go through this as the first one is cached and returned in the static property. Why not
```suggestion
                $dtHelper = new DateTimeHelper($value);
```